integration-cli/docker_cli_proxy_test.go

```diff
@@ -15,9 +16,8 @@ func (s *DockerSuite) TestCliProxyDisableProxyUnixSock(c *check.C) {
 	cmd := exec.Command(dockerBinary, "info")
 	cmd.Env = appendBaseEnv([]string{"HTTP_PROXY=http://127.0.0.1:9999"})
 
-	if out, _, err := runCommandWithOutput(cmd); err != nil {
-		c.Fatal(err, out)
-	}
+	out, _, err := runCommandWithOutput(cmd)
+	c.Assert(err, checker.IsNil, out)
```

Comment:
I'm not sure what will happen to `out` here. The IsNil checker only checks `err`. If `out` is supposed to be printed, I would explicitly put it in a Commentf just to be sure.